src/lang/russian.lng

```diff
@@ -0,0 +1,84 @@
+##grflangid 0x07
+
+STR_GRF_NAME:{TITLE}{VERSION}
+STR_GRF_DESC:{ORANGE}Индустрия, вдохновленная ванилью{BLACK}, с новыми цепочками, рабочими на дорогах, предприятиями вблизи городов и реальными датами внедрения уже в 1700 г.{}{}{SILVER}Требуются автомобили NewGRF. Несовместимо с другими отраслевыми наборами.{}{}{ORANGE}Посетите веб-сайт для получения инструкций.{}{}{BLACK}Кодировано 2TallTyler.
+
+STR_WEBSITE: https: //github.com/2TallTyler/improved_town_industries/blob/master/README.md
```

Comment:
Website URLs should not have a space between the scheme (`https:`) and the double slash (`//`) as that breaks the URL when typed as is.

Reply:
Thanks, missed that one.